pages/docs/error-handling.en-US.mdx

```diff
@@ -48,6 +48,15 @@ const { data, error } = useSWR('/api/user', fetcher)
 // error.status === 403
 ```
 
+You can also customize our `fetcher` with [Axios](https://axios-http.com/docs/intro)
```

Comment:
It should be better to change "our `fetcher`" to "the `fetcher` function", to eliminate the possibility of being interpreted as a different library or disconnected feature.

Reply:
Thank you ! you're right. Updated in #312 8c4ff68f60f44f50cb37ae937b1c536a9a9386cb